### Geofrenzy GfEntThreshold message
#
# A threshold entitlement message specifies an upper and lower threshold
# relevant to the class and entitlement indices.
#
GfEntHeader ent_header      # Entitlement header
float64     threshold_lower # Lower threshold
float64     threshold_upper # Upper threshold
float64     threshold_unit  # Threshold units (RDK what is this field)
